### ISSUE TITLE
Walking NPCs turning when hitting a wall or other NPC.

### ISSUE THREAD

**Issue by Somiaz** [studio-minus/ppg-bugs-and-requests#2945]
### Feature description

Include a toggleable option for when a character bounces off a wall or other entity and to turn around to walk the opposite direction.

### Feature purpose

Would be a nice aesthetic, as well as allowing NPCs to not rub their face across a wall when they have hit a dead end.

**Comment by ghost** [comment]
uh oh flipping request, get ready folks

**Comment by Drogo-man** [comment]
Flipping heck by zooi

**Comment by DindinYT37** [comment]
Duplicate of or related to:
* #483
* #609
* #804
* #916
* #1074
* #1727 
* [card-78571833](https://github.com/studio-minus/ppg-bugs-and-requests/projects/1#card-78571833)
* #2351
* #2729

**Comment by Somiaz** [comment]
Not a flip request! Them ones seem to be requesting a flip when spawning or for other objects. This is flipping NPCs when hitting a wall after walking.

**Comment by ghost** [comment]
> Not a flip request! Them ones seem to be requesting a flip when spawning or for other objects. This is flipping NPCs when hitting a wall after walking.

flipping is flipping

**Comment by Somiaz** [comment]
Yes, but this is while they are walking/moving. I only requested it cause it's annoying to move them over and over as they keep getting stuck against a wall.

|   ←o  - NPC walks towards wall
|o         - NPC walks into wall
|o→     - NPC turns/flips around
|    o→ - NPC continues to walk in opposite direction

**Comment by DindinYT37** [comment]
If flipping itself won't be added, it also won't be added by walking into a wall.
